### PR TITLE
My First portfolio

### PR DESCRIPTION
**Pull request by kwambiee**
To set up my portfolio:
1. I created homepage, style sheet and added linters file.
2. I used template provided from figma to set up my website.
3. I used flexbox to style and position.
4. I uploaded contact images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <nav class="nav-container">
+            <ul class="nav-links">
+                <li class="logo" >
+                    <a href="#">My logo</a>
+                </li>
+                <li class="bars">
+                    <a href="#"><i class="fa fa-bars fa-2x" aria-hidden="true"></i></a>
+                </li>
+            </ul>
+    </nav>
```
Comment (thecodechaser) [inline index.html]: - [x] Please place logo outside of `<nav>` tag, We should place only navbar items and menu options inside `<nav>`, and logo is not part of our navbar.

Comment (thecodechaser) [inline index.html]: - [x] Please place this entire section in `<header>`, This will make sure our code is semantic.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <main class="main-container">
+        <div class="flex-name">
+            <h1>I'm Kwamboka Glad to see you!</h1>
+        </div>
+        <div class="flex-skills">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
```
Comment (thecodechaser) [inline index.html:33]: - [x] Please place some top padding to the hero section and make sure it matches the design.

### Your design

image

### Figma design
image

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+# Tools
+
+Vscode
+Git
+Github
+
+# Author
+
+Github: https://www.github.com/kwambiee
+LinkeddIn: https://www.linkedin.com/in/joy-kwamboka
```
Comment (thecodechaser) [inline README.md:34]: - [ ] Please add other sections like Contributing, Acknowledgments, etc. You can use this [template](https://github.com/microverseinc/readme-template) but customize it for your project.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @kwambiee👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kwambiee** [comment]
@thecodechaser I have amended as according to the changes. Please review

**Comment by thecodechaser** [comment]
> @thecodechaser I have amended as according to the changes. Please review

@kwambiee You have to go for another review from the dashboard, Please submit the same PR link from your dashboard.

**Comment by kwambiee** [comment]
@thecodechaser Hi, I have not been reviewed yet. I wonder what is taking so long.

File: README.md [hunk 4]
```diff
@@ -1 +1,43 @@
-# First-Portfolio
\ No newline at end of file
+# First Portfolio Website
+
+# Built with
```
Comment (vic778) [inline README.md]: - [OPTIONAL] kindly don't forget the importance of the README file. Its biggest importance is the fact that it helps you to explain your code kilometers in words. Be sure to add the description of your project for more understanding.

File: README.md [hunk 5]
```diff
@@ -1 +1,43 @@
...
+
+1. HTML
+2. CSS-Flexbox
+3. Gitlab
+4. Github
+5. VsCode editor
+
+# Getting started
+
+To set up the portfolio;
```
Comment (vic778) [inline README.md:13]: - [OPTIONAL] kindly add all processes for getting started with the project locally. please add the whole process to start the project locally. imagine that someone uninitiated in programming visits your Project and he wants to visualize it in his machine. explain to him how he can do it in this part.

**Review by vic778 — APPROVED** [review]
### STATUS: APPROVED :ok_hand: : 
Hi, @kwambiee, 

Good job so far!
You have made all the changes requested by the previous review. 

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Descriptive commits and correct use of the Git Flow :fire:
- [x] :white_check_mark: page titles :white_check_mark: text headings :white_check_mark: color contrast :white_check_mark: resize :white_check_mark: basic structure of the page 

## Required Changes ♻️
N/A

## Optional suggestions
- `Please remember that it is very important to use Gitflow.`
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### *** Hi @kwambiee *** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mre55 — APPROVED** [review]
**Hi @kwambiee,**

# Status: APPROVED 🏆 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

Alt Text

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**